### DISCUSSION TITLE
How do I add a vcpkg package and have xmake use that?

### DISCUSSION THREAD

**Discussion by smac89** [xmake-io/xmake#4896] [Q&A] [answered]
Using the following:

```lua
add_requires("vcpkg::hello-imgui 1.4.2#1", { alias = "hello_imgui", configs = { features = {"sdl2-binding"}} })
```

When I run `xmake f -vv`, I get the error:

> error: while checking out baseline from commit '44d94c2edbd44f0c01d66c2ad95eb6982a9a61bc', failed to `git show` versions/baseline.json. This may be fixed by fetching commits with `git fetch`.
error: git failed with exit code: (128).
fatal: not a git repository: '.git'
while checking out baseline 44d94c2edbd44f0c01d66c2ad95eb6982a9a61bc

Can someone please help me to understand what the issue might be, and how to fix it?

**Comment by waruqi** [comment, marked as answer]
you can remove version or try to modify baseline. like this https://github.com/xmake-io/xmake/blob/4d4edd5ba69932b2206c891a18dee0f734327767/tests/projects/package/vcpkg_manifest/xmake.lua#L2
